src/enterprise/LICENSE

```diff
@@ -0,0 +1 @@
+Files in this directory ("enterprise") are not under the default repository license. Contact sales<at>defguard.net for further information.
```

Comment:
For enterprise license and usage of this code & binary distribution with this code please contact by email: sales<at>defguard.net